javascript/javascript1/week4/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
```

Comment:
you can eliminate Internet explorer related, rarely any company now supporting version less then 7, 8 etc